### ISSUE TITLE
Source code for prebuild bucketing

### ISSUE THREAD

**Issue by HonyZahy** [happypepper/DeepHoldem#12]
Hi, fantastic job!!!!

Can you commit code how to made bucket static dat file?
I would like to study how to do it.

**Comment by aikupoker** [comment]
There is a way to have prebuild bucketing code? @happypepper 

Thanks a lot!

**Comment by cubesnyc** [comment]
Yes, I would love to see the bucketing code, or any literature or guidance on how to generate it.
